apps/prairielearn/src/lib/editors.js

```diff
@@ -260,7 +260,7 @@ class Editor {
       });
 
       // For each subdirectory, try to find an Info file
-      async.each(directories, async (dir) => {
+      for (const dir of directories) {
```

Comment:
This will be slower (we'll process directories in series rather than in parallel) but will avoid huge fanouts of FS operations, which I think is for the best?

Reply:
As discussed with @mwest1066, I switched back to `async.each` in 22d58c0.